app/views/list/_table.html.erb

```diff
@@ -0,0 +1,18 @@
+<table class="govuk-table">
```

Comment:
this is all unrelated code, I think you've branched off a feature branch rather than master?